### ISSUE TITLE
Bug: TestifyFeatures enabled by default can not be disabled

### ISSUE THREAD

**Issue by DanielJette** [Shopify/android-testify#148]
If you look at the implementation in `TestifyFeatures.isEnabled`, you'll notice that we use an `or` logic on the default value. This means that you can never override the default if it's true.

```kotlin
    internal fun isEnabled(context: Context? = null): Boolean {
        return enabled || context?.isEnabledInManifest ?: enabled
    }
```
https://github.com/Shopify/android-testify/blob/master/Library/src/main/java/com/shopify/testify/TestifyFeatures.kt#L25

**Comment by DanielJette** [comment]
Also, why is `setEnabled` `internal`? Seems that should be a public API

**Comment by DanielJette** [comment]
Fixed in #149
